Marlin/src/lcd/menu/menu_advanced.cpp

```diff
@@ -544,6 +547,10 @@ void menu_backlash();
       EDIT_ITEM_FAST(long5_25, MSG_AMAX_E, &planner.settings.max_acceleration_mm_per_s2[E_AXIS], 100, max_accel_edit_scaled.e, []{ planner.refresh_acceleration_rates(); });
     #endif
 
+    #if ENABLED(DEFAULT_ACCELERATION_SPINDLE)
+      EDIT_ITEM_FAST(long5_25, MSG_A_SPINDLE, &cutter.acceleration_spindle_deg_per_s2, 100, accel_edit_scaled_spindle, []{ });
```

Comment:
Should be something like this to work

    #if ENABLED(ACCELERATION_SPINDLE)
      EDIT_ITEM_FAST(uint16_5, MSG_A_SPINDLE, &cutter.acceleration_spindle_deg_per_s2, 100, accel_edit_spindle, []{ });
    #endif